core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -29,6 +26,35 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
 
   private val cellsNotDone = new AtomicReference[Map[Cell[_, _], Cell[_, _]]](Map())
 
+  /**
+   * Returns a new cell in this HandlerPool.
```

Comment:
I'm not sure we should add these methods to `HandlerPool`, since cells and cell completers are created using a different API. In fact, I think we should use implicit handler pools, since this is a common pattern used also in futures, where the execution context is an implicit parameter normally.

Reply:
In our meeting today, we planned to change this API in different PR that also ensures pool/cell compatibility, so I did not change this for now.